website/screens/components/image/code/ImageCodePage.tsx

```diff
@@ -0,0 +1,187 @@
+import {
+  DxcFlex,
+  DxcTable,
+  DxcLink,
+  DxcParagraph,
+} from "@dxc-technology/halstack-react";
+import Code from "@/common/Code";
+import DocFooter from "@/common/DocFooter";
+import QuickNavContainer from "@/common/QuickNavContainer";
+import QuickNavContainerLayout from "@/common/QuickNavContainerLayout";
+import Example from "@/common/example/Example";
+import HeaderDescriptionCell from "@/common/HeaderDescriptionCell";
+
+const sections = [
+  {
+    title: "Props",
+    content: (
+      <DxcTable>
+        <tr>
+          <th>Name</th>
+          <th>Default</th>
+          <HeaderDescriptionCell>Description</HeaderDescriptionCell>
+        </tr>
+        <tr>
+          <td>alt: string</td>
+          <td></td>
+          <td>
+            Alternative text description displayed when the specified image is not loaded.
+            See{" "}
+            <DxcLink
+              newWindow
+              href="https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/alt"
+            >
+              MDN
+            </DxcLink>{" "}
+            and the{" "}
+            <DxcLink
+              newWindow
+              href="https://www.w3.org/WAI/tutorials/images/decision-tree/"
+            >
+              alt decision tree
+            </DxcLink>{" "}
+            of W3C for further information. This prop is{" "}
+            <strong>required</strong>.
+          </td>
+        </tr>
+        <tr>
+          <td>caption: string</td>
+          <td></td>
+          <td>
+            Image legend with a descriptive purpose. It is placed below the
+            image and is complementary to the <Code>alt</Code> attribute, which
+            is required regardless of the presence of the caption or not.
+          </td>
```

Comment:
I would add here the link to MDN `figcaption` documentation. https://developer.mozilla.org/es/docs/Web/HTML/Element/figcaption

Reply:
Same as above.

Reply:
Agree!